recipes/iqtree/build.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
+if [ "$(uname)" == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7
```

Comment:
Please update this to 10.9 so that it matches the patchfile.

Reply:
It was originally set to 10.9.  You asked to try 10.7 after that build failed.  Do you remember why you wanted it set to 10.7 on Friday?

Reply:
>  Do you remember why you wanted it set to 10.7 on Friday?

Yep! We temporarily tested against 10.7 because we noticed that half the build was using 10.7 while the other was using 10.9 - you changed this env var to 10.7 to see if we could get everything building against the same version of macos. That succeeded in getting everything building against 10.7, but didn't do anything to address the underlying problem. So then the next step was to try to build everything against 10.9. We started that by creating a patch to the CMake config to force that end to use 10.9, next we need to fix up this env var to use 10.9, too.